src/node.js

```diff
@@ -236,7 +257,7 @@ module.exports = class Node {
   setConfig (key, value, done) {
     run(this.exec, ['config', key, value, '--json'], {env: this.env})
       .on('error', done)
-      .on('data', (data) => {})
+      .on('data', () => {})
       .on('end', () => done())
```

Comment:
I believe if this errors, it will call done twice, unless there is a test showing something different